src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+
+        final String Path = "fruitShop.csv";
```

Comment:
Avoid hardcoding file names directly in the code. It's better to pass them as program arguments or read them from a configuration file.